canonical-web-and-design/ubuntu.com
# Add redirect for active directory to discourse topic

## Done

- Add redirect from /desktop/activedirectory to https://discourse.ubuntu.com/t/service-sssd/11579

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/activedirectory
- Should be redirected to the topic: https://discourse.ubuntu.com/t/service-sssd/11579
